app/presenters/export/csv/local_authority_presenter_module.rb

```diff
@@ -11,6 +11,10 @@ def local_authority_name
     @project.local_authority.name
   end
 
+  def local_authority_contact
+    @project.contacts.where(category: "local_authority")
```

Comment:
This will return a `Contact` record, can you return the email address (I presume) of this record. Plus guard for if the record doesn't exist.

Reply:
Have a look at `assigned_to_email` in the ProjectPresenter, and the `main_contact` methods in the same file. In the specs, add contexts for when the contact doesn't exist/is nil.